tests/phpunit/tests/admin/includesScreen.php

```diff
@@ -302,6 +320,9 @@ function test_post_type_edit_collisions() {
 		$this->assertSame( $screen->base, 'post' );
 	}
 
+	/**
+	 * @covers WP_Screen::add_help_tab
```

Comment:
```suggestion
	 * @covers WP_Screen::add_help_tab
	 * @covers WP_Screen:: get_help_tab
	 * @covers WP_Screen:: get_help_tabs
	 * @covers WP_Screen::remove_help_tab
	 * @covers WP_Screen::remove_help_tabs
```

This test is testing all of the help tab(s) methods (doing too much).